projects/Gancio/default.nix

```
@@ -0,0 +1,24 @@
+{
+  pkgs,
+  lib,
+  sources,
+}@args:
+{
+  metadata = {
+    summary = "Gancio Shared agenda for local communities that supports Activity";
```

```suggestion
    summary = "Shared agenda for local communities that supports Activity Pub";
```

From https://nlnet.nl/project/Gancio/